backend/dissemination/templates/search-alert-error.html

```diff
@@ -0,0 +1,8 @@
+{% load humanize %}
+{% load sprite_helper %}
+<div class="usa-alert usa-alert--error">
+    <div class="usa-alert__body">
+        <h2 class="usa-alert__heading">Error</h2>
+        <p>Oops! It looks like some of the information you entered is invalid. Please double-check your information for errors.</p>
```

Comment:
Not sure about "Oops." Might be too friendly for the FAC's voice? @James-Paul-Mason, how do you feel?

We have a similar sort of message here:
https://github.com/GSA-TTS/FAC/blob/53c8abdb5e8c387d7de5ec987d430d96a1b8339f/backend/audit/templates/audit/manage-submission-change-access.html#L34-L36

Reply:
Once we get guidance from James on this I'm good with approving. Works great!